lib/kuzzle/kuzzle.ts

```diff
@@ -79,7 +82,56 @@ Reflect.defineProperty(global, 'kuzzle', {
  * @extends EventEmitter
  */
 class Kuzzle extends KuzzleEventEmitter {
-  constructor (config) {
+  public config: JSONObject;
```

Comment:
Why everything is public? IMHO most of those properties should be private instead